server/pypi/packages/scipy/patches/chaquopy.patch

```diff
@@ -1,43 +1,3 @@
-diff -aur src-orig/pyproject.toml src/pyproject.toml
---- src-orig/pyproject.toml	2023-09-20 11:08:01.936923429 +0200
-+++ src/pyproject.toml	2023-09-21 13:34:47.373780906 +0200
-@@ -14,35 +14,7 @@
-     "Cython>=0.29.18,<3.0",
-     "pybind11>=2.4.3,<2.9.0",
-     "pythran>=0.10.0,<0.11.0",
--
--    # NumPy dependencies - to update these, sync from
--    # https://github.com/scipy/oldest-supported-numpy/, and then
--    # update minimum version to match our install_requires min version
--    # ----------------------------------------------------------------
--
--    # numpy 1.19 was the first minor release to provide aarch64 wheels, but
--    # wheels require fixes contained in numpy 1.19.2
--    "numpy==1.19.2; python_version=='3.8' and platform_machine=='aarch64' and platform_python_implementation != 'PyPy'",
--    # aarch64 for py39 is covered by default requirement below
--
--    # arm64 on Darwin supports Python 3.8 and above requires numpy>=1.20.0
--    "numpy==1.20.0; python_version=='3.8' and platform_machine=='arm64' and platform_system=='Darwin'",
--    "numpy==1.20.0; python_version=='3.9' and platform_machine=='arm64' and platform_system=='Darwin'",
--
--    # Python 3.8 on s390x requires at least 1.17.5, see https://github.com/scipy/oldest-supported-numpy/issues/29
--    "numpy==1.17.5; python_version=='3.8' and platform_machine=='s390x' and platform_python_implementation != 'PyPy'",
--
--    # default numpy requirements
--    "numpy==1.17.3; python_version=='3.8' and (platform_machine!='arm64' or platform_system!='Darwin') and platform_machine!='aarch64' and platform_machine!='s390x' and platform_python_implementation != 'PyPy'",
--    "numpy==1.19.3; python_version=='3.9' and (platform_machine!='arm64' or platform_system!='Darwin') and platform_python_implementation != 'PyPy'",
--    "numpy==1.21.4; python_version=='3.10' and platform_python_implementation != 'PyPy'",
--
--    # For Python versions which aren't yet officially supported,
--    # we specify an unpinned NumPy which allows source distributions
--    # to be used and allows wheels to be used as soon as they
--    # become available.
--    "numpy; python_version>='3.11'",
--    "numpy; python_version>='3.8' and platform_python_implementation=='PyPy'",
-+    "numpy==1.20.3",
```

Comment:
It doesn't matter what version of NumPy it installs in the build environment, because the host environment will take priority through being on the PYTHONPATH.

However, there was an issue with pip attempting to build NumPy from source because it was affected by build-wheel's environment variables; I've fixed this in the most recent commit.